### PR TITLE
Frontier - Add assetmantle

### PR DESCRIPTION
**Pull request by Galadrin**

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -2840,4 +2846,41 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
...
+		rpc: 'https://rpc.assetmantle.one/',
+		rest: 'https://rest.assetmantle.one/',
+		chainId: 'mantle-1',
+		chainName: 'AssetMantle',
+		stakeCurrency: {
+			coinDenom: 'MNTL',
+			coinMinimalDenom: 'umntl',
+			coinDecimals: 6,
+			coinGeckoId: 'asset-mantle',
+			coinImageUrl: window.location.origin + '/public/assets/tokens/mantle.png',
```
Comment (JeremyParish69) [inline src/config.ts:2859]: Please rename the png to 'mntl.png'

File: src/config.ts [hunk 2]
```diff
@@ -2840,4 +2846,41 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
...
+	{
+		rpc: 'https://rpc.assetmantle.one/',
+		rest: 'https://rest.assetmantle.one/',
+		chainId: 'mantle-1',
+		chainName: 'AssetMantle',
+		stakeCurrency: {
+			coinDenom: 'MNTL',
+			coinMinimalDenom: 'umntl',
+			coinDecimals: 6,
+			coinGeckoId: 'asset-mantle',
```
Comment (JeremyParish69) [inline src/config.ts:2858]: Can't find this ID in CoinGecko's coins list v3. Is it coming soon? or could use 'pool:umntl' instead (and define which pool)?

File: src/config.ts [hunk 3]
```diff
@@ -2840,4 +2846,41 @@ export const EmbedChainInfos: ChainInfoWithExplorer[] = [
...
+			coinImageUrl: window.location.origin + '/public/assets/tokens/mantle.png',
+		},
+		bip44: {
+			coinType: 118,
+		},
+		bech32Config: Bech32Address.defaultBech32Config('mantle'),
```
Comment (JeremyParish69) [inline src/config.ts:2864]: Please register 'mantle' on SLIP173
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Comment (avkr003) [reply src/config.ts:2864]: Raised PR for the same
https://github.com/satoshilabs/slips/pull/1308

File: src/config.ts (hunk 2 above)
Comment (Galadrin) [reply src/config.ts:2858]: ticket for listing ongoing,
submit with `assetmantle` in the form

File: src/stores/root.ts [hunk 4]
```diff
@@ -345,6 +345,13 @@ export class RootStore {
 					spotPriceDestDenom: 'uosmo',
 					destCoinId: 'osmosis',
 				},
+				{
+					alternativeCoinId: 'pool:assetmantle',
```
Comment (JeremyParish69) [inline src/stores/root.ts:349]: please change to 'pool:umntl'

**Pull request merged**